src/ring_theory/witt_vector/structure_polynomial.lean

```diff
@@ -317,4 +326,231 @@ begin
   refl
 end
 
+@[simp]
+lemma constant_coeff_witt_structure_rat_zero (Φ : mv_polynomial idx ℚ) :
+  constant_coeff (witt_structure_rat p Φ 0) = constant_coeff Φ :=
+begin
+  rw witt_structure_rat,
+  simp only [bind₁, map_aeval, X_in_terms_of_W_zero, aeval_X, constant_coeff_witt_polynomial,
+    constant_coeff_rename, constant_coeff_comp_algebra_map],
+  exact @aeval_zero' _ _ _ _ _ (algebra.id _) Φ
+end
+
+lemma constant_coeff_witt_structure_rat (Φ : mv_polynomial idx ℚ) (h : constant_coeff Φ = 0) (n : ℕ) :
+  constant_coeff (witt_structure_rat p Φ n) = 0 :=
+begin
+  rw witt_structure_rat,
+  -- we need `eval₂_hom_zero` but it doesn't exist
+  have : (eval₂_hom (ring_hom.id ℚ) (λ (_x : idx), 0)) Φ = constant_coeff Φ :=
+    @aeval_zero' _ _ _ _ _ (algebra.id _) Φ,
```

Comment:
Would it be hard to define `eval₂_hom_zero`? It should also help in the lines below and in `constant_coeff_witt_structure_rat_zero`.

Reply:
Good point. I will add it asap

Reply:
Fixed